<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSBasicFilters</class>
 <widget class="QWidget" name="OBSBasicFilters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>726</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="asyncWidget">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="filtersList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="widget">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="addFilter">
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>Add</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="obs.qrc">
              <normaloff>:/res/images/plus.svg</normaloff>:/res/images/plus.svg</iconset>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="themeID" stdset="0">
             <string>addIconSmall</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeFilter">
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>Remove</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="obs.qrc">
              <normaloff>:/res/images/minus.svg</normaloff>:/res/images/minus.svg</iconset>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="themeID" stdset="0">
             <string>removeIconSmall</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveFilterUp">
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>MoveUp</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="obs.qrc">
              <normaloff>:/res/images/up.svg</normaloff>:/res/images/up.svg</iconset>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="themeID" stdset="0">
             <string>upArrowIconSmall</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveFilterDown">
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>MoveDown</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="obs.qrc">
              <normaloff>:/res/images/down.svg</normaloff>:/res/images/down.svg</iconset>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <property name="themeID" stdset="0">
             <string>downArrowIconSmall</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="asyncToolbarSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="propertiesFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="propertiesLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionRemoveFilter">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/minus.svg</normaloff>:/res/images/minus.svg</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/up.svg</normaloff>:/res/images/up.svg</iconset>
   </property>
   <property name="text">
    <string>Basic.MainMenu.Edit.Order.MoveUp</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="icon">
    <iconset resource="obs.qrc">
     <normaloff>:/res/images/down.svg</normaloff>:/res/images/down.svg</iconset>
   </property>
   <property name="text">
    <string>Basic.MainMenu.Edit.Order.MoveDown</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionRenameFilter">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
 </widget>
 <customwidgets/>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections/>
</ui>
